cpp/notebooks/ros2_subscribers.ipynb

```diff
@@ -1,9 +1,17 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0572b1ad",
+   "metadata": {},
+   "source": [
+    "## Tutorial to use ros2 subscriber"
```

Comment:
Same here: let apurv's pr take care of these. @Apurv354 Please check if these changes need to be added in your PR

Reply:
the changes were added to make the formats of all ros2 examples to be consistent. But, yes, I will remove these for @Apurv354 to take care of these.